src/screens/ProfileScreen/ProfileScreen.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import {
+  FlatList,
+  ListRenderItemInfo,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import {
+  ExperiencesData,
+  ExperiencesDataItemType,
+  MoreSettingItemType,
+  MoreSettings,
+} from '../../assets';
+import {Button, CustomIcon} from '../../components';
+import {fontStyles} from '../../constants';
+import {useThemeColor} from '../../hooks';
+import {Strings} from '../../i18n';
+import {LocaleUtils, horizontalScale, verticalScale} from '../../utils';
+import ExperienceCard from './components/ExperienceCard/ExperienceCard';
+import ProfileHeader from './components/ProfileHeader';
```

Comment:
Update index of components with these new ones

Reply:
Updated indexes.